### ISSUE TITLE
Links from other applications open new windows instead of opening buffers in the existing window

### ISSUE THREAD

**Issue by Kabouik** [atlas-engineer/nyxt#2112]
**Describe the bug**
My `nyxt.desktop` is custom but I use `%u` in the `exec` line and it used to open links from other programs in the existing Nyxt window. With 9691d7b051b31dadec3e22c1e2f7d6f682aa21cc (and perhaps earlier), links are now opened in a new window.

**Precise recipe to reproduce the issue**
Update to no later than 9691d7b051b31dadec3e22c1e2f7d6f682aa21cc, set Nyxt as the default browser in your OS, and open links from other programs.

**Information**
- OS name+version: Solus, rolling
- Graphics card and driver: Intel chipset
- Desktop environment / Window manager name+version: i3
- How you installed Nyxt (Guix pack, package manager, build from source): built from sources
- Information from `show-system-information`:

```
Nyxt version: 2.1.1-1699-g9691d7b0
Renderer version: GI-GTK
Operating system kernel: Linux 5.14.21-210.current
Lisp implementation: SBCL 2.1.10 (Dynamic space size: 3221225472)
Features: (:WEBKIT2 :WEBKIT2-2.34 :WEBKIT2-CORS-ALLOWLIST :WEBKIT2-PASTE-PLAINTEXT
 :WEBKIT2-TRACKING :WEBKIT2-MUTE :WEBKIT2-EMOJI :WEBKIT2-MEDIA
 :WEBKIT2-SANDBOXING :GTK-3-22 :GTK-3-20 :GTK-3-18 :GTK-3-16 :GTK-3-14
 :GTK-3-12 :GTK-3-10 :GTK-3-8 :GTK-3-6 :GTK-3-4 :GTK :GDK-3-22 :GDK-3-20
 :GDK-3-18 :GDK-3-16 :GDK-3-14 :GDK-3-12 :GDK-3-10 :GDK-3-8 :GDK-3-6 :GDK-3-4
 :CAIRO-1-10 :CAIRO-1-12 :GDK-PIXBUF :GLIB-2-30 :GLIB-2-32 :GLIB-2-34
 :GLIB-2-36 :GLIB-2-38 :GLIB-2-40 :GLIB-2-42 :GLIB-2-44 :GLIB-2-46 :GLIB-2-48
 :GLIB-2-50 :GLIB-2-52 :GLIB-2-54 :GLIB-2-56 :GLIB-2-58 :GLIB :NYXT-UNSTABLE
 :NYXT-G9691D7B0 :NYXT-2.1.1 :NYXT-2.1 :NYXT-2 :FSET-EXT-STRINGS
 :CUSTOM-HASH-TABLE-NATIVE :SWANK :PLUMP-UTF-32 :PARENSCRIPT :GLOBAL-VARS
 :DECLARE-TYPES :NAMED-READTABLES :LPARALLEL :21BIT-CHARS :CLOSER-MOP
 :CL-PPCRE-UNICODE :CL-UNICODE :CL-PPCRE :CHUNGA :FLEXI-STREAMS
 :BORDEAUX-THREADS :LPARALLEL.WITH-CLTL2 :LPARALLEL.WITH-CAS
 :LPARALLEL.WITH-STEALING-SCHEDULER :SPLIT-SEQUENCE
 CFFI-FEATURES:FLAT-NAMESPACE CFFI-FEATURES:X86-64 CFFI-FEATURES:UNIX :CFFI
 CFFI-SYS::FLAT-NAMESPACE ALEXANDRIA::SEQUENCE-EMPTYP :FAST-IO-SV :FAST-IO
 :CL-JSON-CLOS :CL-JSON :SBCL-USES-SB-ROTATE-BYTE CHIPZ-SYSTEM:GRAY-STREAMS
 :THREAD-SUPPORT :ASDF3.3 :ASDF3.2 :ASDF3.1 :ASDF3 :ASDF2 :ASDF :OS-UNIX
 :NON-BASE-CHARS-EXIST-P :ASDF-UNICODE :X86-64 :GENCGC :64-BIT :ANSI-CL
 :COMMON-LISP :ELF :IEEE-FLOATING-POINT :LINUX :LITTLE-ENDIAN
 :PACKAGE-LOCAL-NICKNAMES :SB-CORE-COMPRESSION :SB-LDB :SB-PACKAGE-LOCKS
 :SB-THREAD :SB-UNICODE :SBCL :UNIX)

ASDF version: 3.3.1
ASDF registries: (NYXT-SOURCE-REGISTRY ENVIRONMENT-SOURCE-REGISTRY)
Critical dependencies: (/home/mathieu/Projects/common-lisp/nyxt/_build/cl-cffi-gtk/gtk/cl-cffi-gtk.asd
 /home/mathieu/Projects/common-lisp/nyxt/_build/cl-gobject-introspection/cl-gobject-introspection.asd
 /home/mathieu/Projects/common-lisp/nyxt/_build/cl-webkit/webkit2/cl-webkit2.asd)
```

**Comment by Ambrevar** [comment]
Should be fixed with 74c6e88519a299615ef6a9cdcc8bf956a9c7024f.

**Comment by Kabouik** [comment]
Confirmed here, thanks a lot for quickly fixing this.
